jjb/dynamic/scale-ci-pipeline-bm-4.7.yml

```diff
@@ -0,0 +1,345 @@
+    - bool:
+        default: false
+        description: ''
+        name: OPENSHIFTV4_INSTALL_ON_OSP
+    - bool:
+        default: true
+        description: ''
+        name: OPENSHIFTV4_INSTALL_ON_BM
+    - bool:
+        default: false
+        description: ''
+        name: TOOLING
+    - bool:
+        default: false
+        description: ''
+        name: CONFORMANCE
+    - bool:
+        default: true
+        description: ''
+        name: OPENSHIFTv4_SCALE
+    - bool:
+        default: true
+        description: ''
+        name: INGRESS_PERFORMANCE
+    - bool:
+        default: false
+        description: ''
+        name: SERVICES_PER_NAMESPACE
+    - bool:
+        default: false
+        description: ''
+        name: DEPLOYMENTS_PER_NS
+    - bool:
+        default: false
+        description: ''
+        name: PGBENCH_TEST
+    - bool:
+        default: false
+        description: ''
+        name: MONGODB_YCSB_TEST
+    - bool:
+        default: true
+        description: ''
+        name: CLUSTER_DENSITY
+    - bool:
+        default: true
+        description: ''
+        name: KUBELET_DENSITY
+    - bool:
+        default: false
+        description: ''
+        name: KUBELET_DENSITY_LIGHT
+    - bool:
+        default: false
+        description: ''
+        name: NS_PER_CLUSTER
+    - bool:
+        default: false
+        description: ''
+        name: PROMETHEUS_TEST
+    - bool:
+        default: false
+        description: ''
+        name: BYO_SCALE_TEST
+    - bool:
+        default: false
+        description: ''
+        name: LOGGING_SCALE_TEST
+    - bool:
+        default: false
+        description: ''
+        name: BASELINE_SCALE_TEST
+    - bool:
+        default: true
+        description: ''
+        name: UPERF
+    - bool:
+        default: false
+        description: ''
+        name: KRAKEN
+    - bool:
+        default: false
+        description: ''
+        name: SCALE_CI_MS_OSDE2E
+    - bool:
+        default: false
+        description: ''
+        name: SCALE_CI_MS_ROSA
+    - bool:
+        default: false
+        description: 'Caputers Prometheus DB and Cluster information using must-gather.'
+        name: SCALE_CI_DIAGNOSIS
+    - string:
+        default: http://dell-r510-01.perf.lab.eng.rdu2.redhat.com/perfscale_ci/properties/watcher/watcher.properties
+        description: ''
+        name: SCALE_CI_WATCHER_PROPERTY_FILE
+    - string:
+        default: http://dell-r510-01.perf.lab.eng.rdu2.redhat.com/perfscale_ci/properties/aws-next/build-triggers/trigger_4.7.properties
+        description: ''
+        name: SCALE_CI_BUILD_TRACKER_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSTACK_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: BROWBEAT_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSHIFTv3_PROPERTY_FILE
+    - string:
```

Comment:
We might want to set this default to null since it's related to AWS.

Reply:
done